### PR TITLE
Mvs 660 mobile friendly pages and general functionality check

### PR DESCRIPTION
**Pull request by thespaniard514**
## :floppy_disk: [Code Check]
_Make sure you've checked/done the following before making this PR!_
* Does your code build/run?
* Does your design approach make sense?
* Does your work achieve the intention of the card?
* Does your code have any unintentional side effects -- have you tested a full workflow & checked the logic branches you touched?
* Did you make a unit test for your code (as appropriate)?
* Is your branch relatively up to date with the Development branch (within ~2 commits)
* Have you moved the card to "Needs QA" in Jira

## :flipper: [JIRA Sprint Card]

https://mass-immunization-system.atlassian.net/browse/MVS-660

## :newspaper: [Work Description]

General cleanup of code and fixing some user experience things:

- Moved header into it's own partial/component to help de-clutter our App.vue file
- Fixed the header text getting smaller for xs screens
- added some padding at the bottom of our main card so that the buttons don't cut off the bottom
- made a generic utility for storing common validation rules functions to keep things DRY
- added some autocomplete attributes to our address fields so browsers can autocomplete better
- Default language to English as that is the most common

NOTE! Sorry for the formatting changes, my local beautify tool seems to be different than yours.

## :vertical_traffic_light: [Testing/QA Notes]

As most of look and feel, no functional unit tests were needed. This was inspected visually using different browsers and simulating phones and ipads.

### CODE REVIEW

**Review by axn9t9 — CHANGES_REQUESTED** [review]

File: PatientRegistrationApp/src/App.vue [hunk 1]
```diff
@@ -2,34 +2,11 @@
   <v-app id="Patient-registration">
     <v-main>
       <v-container fill-height fluid>
+        <Header />
         <v-layout justify-center align-center>
-          <v-card flat height="101%" width="100%">
+          <v-card flat height="100%" width="100%">
+            <!-- TODO: move stepper to new page, one for each flow? (or combine flows?) -->
             <v-stepper v-model="page" class="elevation-0">
-              <v-toolbar flat height="80em" color="primary" dark>
```
[the remark this answers is not in the document]
Comment (thespaniard514) [reply PatientRegistrationApp/src/App.vue:8]: Good catch, I wasn't testing at very large widths. I just changed to using row and col so we can take advantage of the grid layout in vuetify

File: PatientRegistrationApp/src/App.vue [hunk 2]
```diff
@@ -2,34 +2,11 @@
   <v-app id="Patient-registration">
     <v-main>
       <v-container fill-height fluid>
+        <Header />
         <v-layout justify-center align-center>
-          <v-card flat height="101%" width="100%">
+          <v-card flat height="100%" width="100%">
+            <!-- TODO: move stepper to new page, one for each flow? (or combine flows?) -->
```
[the remark this answers is not in the document]
Comment (thespaniard514) [reply PatientRegistrationApp/src/App.vue:9]: I didn't want to do too much work in this PR so I was just taking some notes here.  Some cleanup could happen after we get complete functionality end to end.

**Comment by thespaniard514** [comment]
Don't be alarmed with the full change in App.vue, it is mostly a shift over one tab in styling because I added an extra element `<v-col cols="12">`

**Review by dmcook5208 — APPROVED** [review]
Reviewed in Safari using iPhone SE (smallest), iPad, and 1920 x 1080 resolutions.  This all looks great!

**Review by axn9t9 — DISMISSED** [review]

**Review by dmcook5208 — APPROVED** [review]
Great!  The logo is right-side-up in Safari and Chrome! :)

**Pull request merged**